[DRK] first draft of advanced guide (#2864)
The-Balance-FFXIV/balance-static: content/jobs/tanks/dark-knight/advanced-guide.md

For example, if you have 9600 MP and Blood Weapon active, you can use Disesteem, immediately weave Edge of Shadow, and not overcap MP, as the Edge of Shadow will spend MP before Disesteem (and the Blood Weapon buff that modified it to give you +600 MP) applies. If you were to use Hard Slash in the same situation, you would overcap MP.

Another ability where the application delay is worth knowing is Carve and Spit, which has an application delay of 1.47s, similarly allowing an Edge of Shadow weave before the MP gain applies.

This can come up in practical situations if you're going into burst with an especially high amount of MP. In these situations, you want to save Disesteem for the GCD where you'd normally overcap. If you have, for example, 9000 MP and a Dark Arts proc, then with Blood Weapon and Delirium up, using Disesteem -> Edge of Shadow -> Scarlet Delirium -> Edge of Shadow would overcap MP, whereas Scarlet Delirium -> Edge of Shadow -> Disesteem -> Edge of Shadow would not.

> # Salted Earth and Buff Windows
> 
> Salted Earth is a weird in that it has a 90s cooldown, unlike most other damaging cooldowns in the game. Regardless, it's best to use Salted Earth off cooldown rather than save it for buffs, since more uses of the ability will always trump less uses in buff windows. This only changes when you know the killtime and you know you wouldn't lose a use to hold the ability for buffs. For example, if you're killing the boss after 8:30, then you wouldn't lose a use of Salted Earth if you held the one you'd use around 1:30 to the 2m burst window. This also shifts the 7:30 Salted Earth to the 8m window, allowing us to pot in the 2m and 8m burst window (instead of the 0m and 6m burst window) without losing abilities under pot.

Reviewer: ```suggestion
# Salted Earth and Buff Windows

Salted Earth is unusual in that it has a 90 second cooldown, unlike most other damaging cooldowns in the game, and so it does not always line up with buff windows, which generally happen every 120 seconds.
Regardless, it is usually best to use Salted Earth off cooldown rather than to save it for buffs, since more uses of the ability will almost always be better than fewer uses in buff windows.

For a simple example, in a fight that lasts 6:30, using Salted Earth on cooldown gives 5 full usages at (approximately) 0:00, 1:30, 3:00, 4:30, and 6:00.
Delaying Salted Earth to fit in buffs gives 4 usages, at 0:00, 2:00, 4:00, and 6:00.
The usages at 0:00 and 6:00 are the same in both cases, and so can be ignored for a comparison.
With the delay, you trade three unbuffed usages (1:30, 3:00, 4:30) of Salted Earth for two buffed usages (2:00, 4:00).
As such, for this delay to be a gain, a buffed usage needs to be at least 1.5 times as strong as an unbuffed usage, which essentially never happens with a normal group composition.
If your group has particularly many buffs, you may find it worthwhile to double check if the buffs reach a 1.5 multiplier.

If you know your killtime (when the boss will die) or a phase timing (when the boss will disappear for a long time, essentially resetting your cooldowns), you may be able to delay Salted Earth to gain more buffed usages without losing overall usages, for an overall damage gain.
For example, with an 8:30 killtime, delaying your 7:30 Salted Earth to 8:00 exchanges the unbuffed 7:30 usage for a buffed 8:00 usage.
Alternately, delaying the 1:30 Salted Earth (and all subsequent ones) by 30 seconds moves the 1:30 and 7:30 usages into buffs at 2:00 and 8:00, respectively, in exchange for moving the 6:00 usage (in buffs) to 6:30 (out of buffs).
Both of these shifts have the same effect on the number of Salted Earth in buff windows, but the second option lets you have Salted Earth in 2:00 and 8:00 potions, rather than the standard 0:00 and 6:00 potion timing.
```

This mostly adds a bit more detail to examples/a bit of maths, but also reminds me that potion timing might be good to add as a section somewhere (and then this last note about 2/8 potions can refer back to that).

> ## Salted Earth Buff Snapshotting
> 
> Salted Earth snapshots all buffs on the player when cast, but checks damage modifying debuffs (Chain Stratagem, etc) on targets every tick. Because it is a ground effect ability, Salted Earth also gains a free tick on all targets in range immediately when cast, giving it 300 total potential potency (which is slightly increased by the skill speed DoT scalar).
> 
> Salt and Darkness, Salted Earth's follow-up ability, snapshots buffs independently of Salted Earth, and snapshots both buffs on the player and damage modifying debuffs when used, separately to Salted Earth.

Reviewer: ```suggestion
## Salted Earth Buff Snapshotting

Salted Earth snapshots all buffs on the player when cast, but checks damage modifying debuffs (e.g., Chain Stratagem, Dokumori, etc.) on targets every time it ticks for damage.
Because it is a ground effect ability, Salted Earth also gains a free tick on all targets in range immediately when cast, giving it 300 total potential potency (which is slightly increased by the skill speed DoT scalar).

Salt and Darkness, Salted Earth's follow-up ability, snapshots buffs independently of Salted Earth, and snapshots both buffs on the player and damage modifying debuffs when it is used, separately from Salted Earth.
```

# How Does Living Shadow Work Exactly?

> Living Shadow is a very unique ability. Its abilities work uniquely, and each of them has unique strength scaling and an alternate weapon damage scaling. This section goes into detail to provide exactly how it works, and the formulae that differ between Living Shadow's abilities and Dark Knight's normal abilities.

Reviewer: ```suggestion
Living Shadow is a very unique ability.
Esteem abilities work uniquely, and each of them has unique strength scaling and an alternate weapon damage scaling.
This section provides details on exactly how it works, and the formulae that differ between Living Shadow's abilities and Dark Knight's normal abilities.
```
I'm assuming the reuse of "unique" is intended for effect, but it reads a bit strangely at first glance.

## Living Shadow's Abilities and 'Rotation'

Living Shadow spawns Esteem, which, after a ~6.8 second spawn delay, will perform six abilities, with a ~2.18 second delay between each.

> The abilities Esteem uses do share animations with the player's abilities, but are not those abilities. Esteem's abilities do not share potencies with the equivalent ability for the player, and have their own own potencies. For example, if Bloodspiller gets a potency buff, then Living Shadow's Bloodspiller does not gain the same potency buff.
> 
> At level 100, Living Shadow performs the following abilities, in sequence, totaling 2450 potency. It may start over if the target moves out of range, but six attacks is always the limit:

Reviewer: ```suggestion
The abilities Esteem uses share animations and areas of effect with the player's abilities, but are not the same as those abilities. Esteem's abilities do not share potencies with the equivalent ability for the player, and have their own own potencies. For example, if Bloodspiller gets a potency buff, then Living Shadow's Bloodspiller does not gain the same potency buff.

At level 100, Living Shadow performs the following abilities, in sequence, totaling 2450 potency. It may start over if the target moves out of range, but always stops after executing six abilities:
```

Living Shadow's base strength is calculated differently to the player. In particular:

- Living Shadow's strength does not include the party strength bonus for having different roles in your party (with all roles, this is a usually a 5% boost for the player)

- Living Shadow does not use your base stats (e.g. 23 base strength that races like The Lost have), and instead uses a base strength of 2. Effectively, this means strength is lowered by around 20, depending on your race (21 for max strength races like Xaela and The Lost)

- Living Shadow's strength includes all strength bonuses from gear, as normal

To be more specific, Living Shadow's base strength (before strength buffs like potions) is calculated by the following formula (where `baseStrength` is `440` at level 100):

`baseStrength + strengthProvidedByGear + 2`

Living Shadow's weapon damage multplier is calculated differently to the player. In particular:

- While players would use a weapon damage multplier of `115`, Living Shadow uses a weapon damage multiplier of `100`

- This is typical for all 'pet actions', and generally true for any damage that does not come 'from' the player

> This changes the formula from the following:

Reviewer: The formula for what? I guess this is the damage scalar from strength + WD, but I'm not sure how to express that succinctly. Maybe
```
This changes the formula for damage scaling with strength and weapon damage from the following:
```
?

Author: This is the weapon damage multiplier -- I updated the text to be clearer.
